app/controllers/schools/add_participants_controller.rb

```diff
@@ -42,6 +42,20 @@ def start
       end
     end
 
+    def transfer
+      if form.complete_step(:transfer, form_params)
+        if form.transfer == "true"
```

Comment:
This might be nicer if it was encapsulated in the form so we could call
`if form.transfer?` but I don't want to hold this up